src/app/user/attivita/page.tsx

```diff
@@ -1,12 +1,13 @@
 import ActivitiesTable from "@/components/TableActivities";
-import { getAthleteActivies, getAthleteStats } from "@/functions";
+import { getAthleteActivities, getAthleteStats } from "@/functions";
+import { ActivityStats } from "@/models/activityStats";
 import { Session } from "@/models/session";
 import { cookies } from "next/dist/client/components/headers";
 
 export default async function Attivita() {
 
-    const stats = await getStats();
-    const activities = await getActivies();
+    const stats: ActivityStats = await getStats();
+    const activities = await getActivities();
```

Comment:
Potremmo aggiungere il tipo, in questo modo se cambia il tipo che restituisce getActivities viene evidenziato un errore